pages/table/sticky-header-with-actions.page.tsx

```diff
@@ -20,29 +21,31 @@ export default function () {
   return (
     <>
       <h1>Table with a sticky header and actions</h1>
-      <div style={{ height: '400px', width: '500px', overflow: 'auto', padding: '0px 1px' }} id="scroll-container">
-        <div style={{ height: '100px' }} />
-        <Table<Instance>
-          ariaLabels={selectionLabels}
-          selectionType="multi"
-          stickyColumns={{ last: 1 }}
-          header={
-            <Header
-              actions={
-                <ButtonDropdown data-test-id="actions-button" items={dropdownItems}>
-                  Actions
-                </ButtonDropdown>
-              }
-              headingTagOverride="h1"
-            >
-              Instances
-            </Header>
-          }
-          columnDefinitions={columnsConfig}
-          items={generateItems(1)}
-          stickyHeader={true}
-        />
-      </div>
+      <ScreenshotArea>
+        <div style={{ height: '400px', width: '500px', overflow: 'auto', padding: '0px 1px' }} id="scroll-container">
+          <div style={{ height: '100px' }} />
+          <Table<Instance>
+            ariaLabels={selectionLabels}
+            selectionType="multi"
+            stickyColumns={{ last: 1 }}
+            header={
+              <Header
+                actions={
+                  <ButtonDropdown data-test-gid="actions-button" items={dropdownItems}>
```

Comment:
Is this an intentional change or a typo?

Reply:
Oops, nice catch. Fixed